lib/smart_proxy_container_gateway/container_gateway.rb

```diff
@@ -25,6 +25,12 @@ class Plugin < ::Proxy::Plugin
       validate :pulp_endpoint, url: true
 
       rackup_path File.join(__dir__, 'container_gateway_http_config.ru')
+
+      def load_dependency_injection_wirings(container, settings)
```

Comment:
Oops, this is a DSL, not a separate class:
```suggestion
      load_dependency_injection_wirings do |container, settings|
```

Reply:
Very strange, for some reason, this dependency injecting causes container gateway endpoints to return 404s. Something is bugging Sinatra.

Reply:
Actually, it seems to be killing the entire container gateway plugin, after refreshing the feature has disappeared. Interesting ...

Reply:
`Disabling all modules in the group ['container_gateway'] due to a failure in one of them: uninitialized constant Proxy::Database `, okay, getting closer ...

Reply:
Ah of course, Database was module instead of a class. Things got easier to debug after I found out more logs were available on the smart proxy in the UI.